dht_bootstrap.go

```diff
@@ -252,14 +47,14 @@ func (dht *IpfsDHT) Bootstrap(_ context.Context) error {
 // The returned channel will block until the refresh finishes, then yield the
 // error and close. The channel is buffered and safe to ignore.
 func (dht *IpfsDHT) RefreshRoutingTable() <-chan error {
-	res := make(chan error, 1)
-	select {
-	// FIXME: this can block. Ideally, we'd return a channel without blocking.
-	// https://github.com/libp2p/go-libp2p-kad-dht/issues/609
```

Comment:
nit: I don't have strong feelings about having FIXMEs like this in the code. However, this issue still remains so I'd leave the comment. Your call though

Reply:
I've added this back.